app/controllers/subscription_controller.rb

```diff
@@ -173,7 +173,7 @@ def multiple_add
   private
 
   def tag_selection_more_than_zero?
```

Comment:
I worry about grabbing a parameter directly from a private method. I guess I'd prefer to have it passed in like this:

```suggestion
  def tag_selection_more_than_zero?(tag_id)
```
Then we can change it above too, pass the params[:tid] in when we call `tag_selection_more_than_zero?(params[:tid])`, how does that sound? Thanks!